tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="cp319" name="Cellprofiler" version="19.05">
+  <description>Run a CellProfiler v3.1.9 pipeline</description>
+  <requirements>
+     <requirement type="package" version="3.1.9">cellprofiler</requirement>
+  </requirements>
+  <configfiles>
+<configfile name="script_file">
+mkdir `pwd`/input
+mkdir `pwd`/output
+#for $i,$filename in enumerate($inputType.inputdir):
+ln -s $filename `pwd`/input/${filename.element_identifier}.${filename.ext}
```

Comment:
* remove pwd
* the $filename should be in single-quotes

Reply:
updated